### PR TITLE
navbar functionality added styling

### PR DESCRIPTION
**Pull request by MHavrilak**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -40,12 +42,12 @@ h2 {
 
 #start {
     align-self: auto;
-    margin-left: 45%;
     color: #00d1b2, #fff;
 }
 
 button {
     color:#00d1b2;
+    margin: 10px
```
Comment (kshep425) [inline assets/css/style.css]: add semi colon

File: index.html [hunk 2]
```diff
@@ -35,16 +35,17 @@ <h2 class="subtitle">
         <nav class="navbar is-primary" role="navigate" aria-label="main-navigation">
                     <div class="navbar-brand">
                         <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false"
-                            data-target="navBasicExample">
+                            data-target="navBasic">
                             <span aria-hidden="true"></span>
```
Comment (kshep425) [inline index.html:41]: Should anything be in these spans?

File: index.html [hunk 3]
```diff
@@ -35,16 +35,17 @@ <h2 class="subtitle">
...
                             <span aria-hidden="true"></span>
                             <span aria-hidden="true"></span>
                             
                         </a>
                     </div>
-                    <div id="navbarBasicExample" class="navbar-menu is active">
+                    <div id="navbarBasic" class="navbar-menu is active">
```
Comment (kshep425) [inline index.html:47]: Can you change id's to be underlined?

File: index.html [hunk 4]
```diff
@@ -80,7 +81,7 @@ <h2 class="subtitle">
             </select>
             <button type="submit" value="Save" id="save_profile">Save</button>
         </form>
-        <button class="button is primary is rounded" type="input" id="start">Start Workout!</button>
+        <button class="button is-primary is-outlined is-rounded" type="input" id="start">Start Workout!</button>
```
Comment (kshep425) [inline index.html:86]: Very nice change!

**Review by kshep425 — CHANGES_REQUESTED** [review]
Can you make the updates?  Then I'll approve it.

File: assets/css/style.css (hunk 1 above)
Comment (MHavrilak) [reply assets/css/style.css]: fixed

**Review by kshep425 — COMMENTED** [review]
I don't see the changes.  You may need to push and do another pull request.

**Review by kshep425 — APPROVED** [review]

**Pull request merged**